### ISSUE TITLE
Keyboard navigation does not work while processing queue

### ISSUE THREAD

**Issue by magnusakselvoll** [magnusakselvoll/photo-booth-take-two#55]
While processing the queue, left right should still work. Any keyboard command such as random, skip 10, etc. should completely invalidate the queue.
